.gitignore

```diff
@@ -7,4 +7,5 @@ TODOs.md
 temp
 explorations
 .idea
-*.local
\ No newline at end of file
+*.local
+package-lock.json
```

Comment:
Don't add your env specific changes to `.gitignore`. The project uses Yarn, and includes `yarn.lock`.